x-pack/filebeat/module/sophos/xg/ingest/pipeline.yml

```diff
@@ -28,6 +28,13 @@ processors:
 - set:
     field: _temp_.time
     value: "{{sophos.xg.date}} {{sophos.xg.time}}"
+    if: "ctx?.sophos?.xg?.date != null && ctx?.sophos?.xg?.time != null"
+
+- set:
+    field: _temp_.time
+    value: "{{sophos.xg.timestamp}}"
```

Comment:
This is causing the tests to fail because the `_temp_.time` field can now be a UNIX timestamp in addition to a textual datetime.

You should add `UNIX` to the formats section of the two date processors below, so that it can parse this kind of timestamps.